### PR TITLE
Initial code with unit testing

### PR DESCRIPTION
**Pull request by herpiko**
### Usage
- Edit `conf.json`, change `hostname` value to the dovecot's hostname.
- `HOST=hostname DB=dbname node index.js /path/to/dovecot/log`
- Set the command above as daily cron.
### Test

`npm run test`

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+      console.log(err);
+      process.exit();
+    }
+  });
+}
+mongoose.connection.once('open', function(){
+  mongoose.connection.db.collection('users', start);
+});
+
+var prepare = function(cb) {
```
Comment (mdamt) [inline index.js]: Let's have this as a promise

File: index.js [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+          if (err) {
+            console.log(err);
+          }
+          count++;
+          console.log(count);
+          if (count >= totalLines) {
+            console.log('Done!');
+            console.log(notFound);
+            process.exit();
+          }
```
Comment (mdamt) [inline index.js]: Take some of these out/or make them as promises.

Lot's of callback here.

File: test/import.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        if (err) {
+          throw err;
+        }
+        if (!domainId) {
+          domainId = mongoose.Types.ObjectId(result.ops[0]._id);
+        }
+        if (domains.length-1 == i) {
+          mongoose.connection.db.collection('users', function(err, userCol){
+		        if (err) {
+              throw err;
```
Comment (mdamt) [inline test/import.js:25]: Indentation

File: test/import.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+		        }
+            for (var i in users) {
```
Comment (mdamt) [inline test/import.js]: ES6

**Comment by emhaye** [comment]
Ok :+1:

**Pull request merged**